assignments/assignment01.rb

```diff
@@ -80,6 +84,8 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
+  @s = s.downcase.gsub(/\s+/m, '').gsub(/,/, '')
```

Comment:
Why did you use an instance variable @s instead of creating a new local variable? Other than that, the code looks correct.